atd-toolbox/data_model/python/post_migration_scripts/notes_recs_crash_pk_backfill.sql

```diff
@@ -0,0 +1,34 @@
+-- Backfill crash notes with the crash_pk
+UPDATE
+  crash_notes
+SET
+  crash_pk=crashes.id
+FROM
+  crashes
+WHERE
+  crash_notes.atd_txdot_crashes_crash_id=crashes.cris_crash_id;
+
+-- Backfill crash recommendations with the crash_pk
+UPDATE
+  recommendations
+SET
+  crash_pk=crashes.id
+FROM
+  crashes
+WHERE
+  recommendations.atd_txdot_crashes_crash_id=crashes.cris_crash_id;
+
+-- This is a crufty crash that CRIS has since deleted, it is a dupe of cris crash id 19854368
+DELETE FROM recommendations WHERE atd_txdot_crashes_crash_id = 19724784;
+
+DELETE FROM crash_notes WHERE atd_txdot_crashes_crash_id = 19724784;
+
+-- Drop old columns
+ALTER TABLE recommendations DROP COLUMN atd_txdot_crashes_crash_id;
+
+ALTER TABLE crash_notes DROP COLUMN atd_txdot_crashes_crash_id;
+
+-- Each record must always have an associated crash_pk
+ALTER TABLE recommendations ALTER COLUMN crash_pk SET NOT NULL;
+
+ALTER TABLE crash_notes ALTER COLUMN crash_pk SET NOT NULL;
```

Comment:
i think it would be best to execute these two `SET NOT NULLS` before we drop the `atd_txdot_crashes_crash_id` column, that way we know 100% of records have been assigned a new ID before we drop the old data.